<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1580</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>150</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1571</width>
      <height>761</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Local interaction information:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="serEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="recChooseFile">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>保存文件路径
Save File Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="recFilePath">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>D:/rec.bin</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="ListenButton">
           <property name="text">
            <string>启动监听
Listen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>本地IP地址:
Host IP Address</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="hostIPNum">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>端口
PORT:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="serPort">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="ClientconnectServer">
             <property name="font">
              <font>
               <family>Agency FB</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>连接TCP服务器
Connect TCP Server</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="disClientconnectServer">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Agency FB</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>断开TCP连接
Disconnect TCP</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="font">
              <font>
               <family>Agency FB</family>
               <pointsize>12</pointsize>
               <underline>false</underline>
              </font>
             </property>
             <property name="text">
              <string>进入升级模式
Enter Bootloader</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="CannelUpdate">
             <property name="font">
              <font>
               <family>Agency FB</family>
               <pointsize>12</pointsize>
               <underline>false</underline>
              </font>
             </property>
             <property name="text">
              <string>退出升级模式
ESC Bootloader</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_21">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Device interaction information:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>设备IP地址:
Device IP Address</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="serIPNum">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>端口
PORT:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="serPortNum">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="GetNewFile">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>获取最新文件
Get New Bin File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ChooseFile">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>选择文件
File Choose</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FilePath">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>D:/5G_VER_D2AS.bin</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>升级进度
Upgrade Progress:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="UpdateCode">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>16</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>确认升级
Confirm the upgrade</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="recFilePath_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>740</y>
      <width>664</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D:/rec.bin</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
